docs/reference/api/docker_remote_api.md

```diff
@@ -107,12 +107,11 @@ This section lists each version from latest to oldest.  Each listing includes a
 * `POST /volumes/create` to create a volume.
 * `GET /volumes/(name)` get low-level information about a volume.
 * `DELETE /volumes/(name)`remove a volume with the specified name.
-* `VolumeDriver` has been moved from config to hostConfig to make the configuration portable.
-* `GET /images/(name)/json` now returns information about tags and digests of the image.
+* `VolumeDriver` was moved from `config` to `HostConfig` to make the configuration portable.
+* `GET /images/(name)/json` now returns information about image tags.
```

Comment:
Oh, I see this was suggested by @moxiegirl, but this line deliberately mentioned "tags" and "digests", because both are included (`RepoTags`, `RepoDigests`)

@moxiegirl any reason for dropping "digests"?